src/alert.cpp

```diff
@@ -1965,21 +1966,16 @@ namespace libtorrent
 	std::vector<tcp::endpoint> dht_get_peers_reply_alert::peers() const
```

Comment:
historical reasons...coming from the DHT, that does not make sense, right? Do you have an alternative in mind?